README.md

```diff
@@ -17,7 +16,9 @@ The library includes tooling for:
 - Input Validations
     - Using [JPA Validation](https://www.baeldung.com/spring-boot-bean-validation)
 - Address Validation using [Smarty](https://smarty.com/)
-- A set of [Thymeleaf fragments](https://github.com/codeforamerica/form-flow/tree/main/src/main/resources/templates/fragments) that create a library of reusable HTML components for Inputs, Screens, Forms, etc.
+- A set
```

Comment:
So much of this is just intellij styling this.  Not sure why it wasn't already styled correctly.